media-proxy/src/rdma_session.c

```diff
@@ -345,9 +340,19 @@ static void *rx_rdma_frame_thread(void *arg)
     libfabric_ctx *rdma_ctx = s_ctx->rdma_ctx;
     ep_ctx_t *cp_ctx = s_ctx->ep_ctx;
 
-    printf("%s(%d), start\n", __func__, s_ctx->idx);
+    while (!atomic_load_explicit(&s_ctx->shm_ready, memory_order_acquire) && !s_ctx->stop) {
+        DEBUG("%s memif not ready\n", __func__);
```

Comment:
I suggest to avoid printing that every 1ms.
```suggestion
```

Reply:
ok, I will remove it entirely as a log is printed in places where the variable is changed.